osu.Game.Tests/Visual/Online/TestSceneBeatmapSetOverlay.cs

```diff
@@ -246,6 +246,17 @@ public void TestExplicitBeatmap()
             });
         }
 
+        [Test]
+        public void TestFeaturedBeatmap()
+        {
+            AddStep("show featured map", () =>
+            {
+                var beatmapSet = CreateBeatmap(Ruleset.Value).BeatmapInfo.BeatmapSet;
+                beatmapSet.OnlineInfo.TrackId = 1;
+                overlay.ShowBeatmapSet(beatmapSet);
```

Comment:
If this test is ran after `TestExplicitBeatmap()` (i.e. in test browser) then nothing will actually happen because the beatmap won't change value. See implementation of `ShowBeatmapSet()` and `BeatmapSetInfo` equality.

Can be fixed by assigning an unique ID to both or something of the sort.

Reply:
Fixed by using a random ID (reloading the overlay when one step is executed multiple times).